component-library/shared/styles/global.scss

```diff
@@ -2,7 +2,9 @@
 // as well as in the component browser and screenshots.
 
 html, body {
-    
+	h1, h2, h3, h4 {
```

Comment:
Think this code should be moved to assets style sheet. I am also wondering about the code below this. This .component {} not sure if this is needed.

Reply:
Yep this was put in as example usage and can be removed. 

Do you have a suggestion of how else we would apply the background color variant to all components? I used a single class so it doesn't need to be re-implemented with each component. All components have a dark/light variant toggle so there needs to be a way for the user to define those colors, and then a simple way to apply them to each component based on the chosen variant.

Reply:
I've made some improvements based on James' suggestion below

Reply:
@Yoda-Soda can you please check this again when possible